components/google_sign_in/common/features.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_GOOGLE_SIGN_IN_COMMON_FEATURES_H_
+#define BRAVE_COMPONENTS_GOOGLE_SIGN_IN_COMMON_FEATURES_H_
+
+#include "base/feature_list.h"
+
+namespace google_sign_in::features {
+
+extern const base::Feature kBraveGoogleSignIn;
```

Comment:
the feature name should include Permission, Google and SignIn.

`GoogleSignInPermission` for ex.